### PR TITLE
Use less broken advanced example

### PR DESCRIPTION
**Pull request by FossPrime**
JSFiddle is being JSfiddle, it's not displaying the original example.

### CODE REVIEW

**Comment by MarsOrBust24** [comment]
Addresses #78

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Besides the component itself, the module also includes a `getDocument` function
 
 [Basic Usage Demo (JSFiddle)](https://jsfiddle.net/hrynko/ct6p8r7k)
 
-[Advanced Usage Demo (JSFiddle)](https://jsfiddle.net/hrynko/we7p5uq4)
+[Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
```
Comment (hrynko) [inline README.md:114]:
```suggestion
[Advanced Usage Demo (JSFiddle)](https://jsfiddle.net/hrynko/we7p5uq4)

[Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
```

Let's keep the JSFiddle examples for now and add StackBlitz as a complement

**Review by hrynko — CHANGES_REQUESTED** [review]
Hi @FossPrime,

Thanks for opening this PR. Please consider the following suggestion.

**Comment by FossPrime** [comment]
I've confirmed the example works on Firefox and Chrome.

With a package.json under demo, this could easily run directly from the git repo. stackblitz.com/fork/github/hrynko/vue-pdf-embed/tree/master/demo

See vite.new/vue , vite.new , vite.new/vanilla-ts and vite.new/vue-ts

**Comment by hrynko** [comment]
I see your point, but the JSFiddle example is not broken. It still works in some environments. Thus, I would keep both examples, so that the one can choose the preferred editor.

**Comment by FossPrime** [comment]
I can confirm the JSFiddle Advanced example is now working again in Firefox and Chrome. Stackblitz Classic's WebContainers are a good runtime, but it is not a good editor for Vue projects. Stackblitz CloudFlow is good for running and editing, but requires a special git repo and is not usable in incognito.

File: README.md [hunk 2]
```diff
@@ -113,6 +113,10 @@ Besides the component itself, the module also includes a `getDocument` function
 
 [Advanced Usage Demo (JSFiddle)](https://jsfiddle.net/hrynko/we7p5uq4)
 
+[Advanced Usage Demo (JSFiddle)](https://jsfiddle.net/hrynko/we7p5uq4)
+
+[Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
```
Comment (hrynko) [inline README.md:118]:
```suggestion
[Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
```

**Comment by hrynko** [comment]
Thanks, I appreciate it 👍

**Pull request merged**